### DISCUSSION TITLE
How to always show a module?

### DISCUSSION THREAD

**Discussion by raffaelemancuso** [Alexays/Waybar#1992] [Q&A] [answered]
How to always show a module no matter what?

For instance, if the network is not available, the network module disappears from the bar.

I want to to still appear, with an error or whatever, since then I can bind a mouse click that will check the connection and restore it if it doesn't work.

**Comment by miranyx** [comment, marked as answer]
For the network module you can use `"format-disconnected": "Disconnected  "`

**Reply by raffaelemancuso** [reply]
Thanks!
